systems/controllers/iiwa_position_controller.cc

```diff
@@ -0,0 +1,86 @@
+#define NUM_JOINTS 7
+#define K_P 50
+#define K_OMEGA 50
+#define WORLDFRAME_ID 0
+#define ENDEFFECTOR_FRAME_ID 10
+
+#include "systems/controllers/iiwa_position_controller.h"
+
+namespace dairlib{
+namespace systems{
+
+KukaIiwaPositionController::KukaIiwaPositionController()
```

Comment:
This constructor should take, as arguments: the `RigidBodyTree`, the gains, and the end effector frame ID. The world frame can be safely assumed to be zero. 

As discussed, it's not iiwa specific, so the name should reflect that. Suggest `EndEffectorPositionController`